asn1sccserviceprovider.cpp

```diff
@@ -145,8 +145,9 @@ void Asn1SccServiceProvider::loadServiceParams()
 void Asn1SccServiceProvider::onProcessFinished(int exitCode, QProcess::ExitStatus exitStatus)
 {
     Q_UNUSED(exitCode);
+    Q_UNUSED(exitStatus);
 
-    if (!m_terminating && exitStatus != QProcess::NormalExit) {
+    if (!m_terminating) {
```

Comment:
yep. Also: if watchdog stops asn1scc.d it does exit gracefully, so again - `QProcess::NormalExit` :)

this can even fix the bug we have in current sprint